src/main/java/com/github/jbellis/jvector/vector/VectorizationProvider.java

```diff
@@ -22,7 +22,9 @@
 import java.security.PrivilegedAction;
 import java.util.Locale;
 import java.util.Objects;
-import java.util.logging.Logger;
+
+import org.slf4j.Logger;
```

Comment:
let's not do this, it's just at startup and the hdf5 dependency is going to be moved to a separate module so core will stay clean

Reply:
Why? do you not like slf4j? I think it's quite good compared to standard.